MetaMask/metamask-mobile
# Cleanup approve transaction review

Fixes: #6159

The task was about extracting out tooltip component from ApproveTransactionReview component but I found that these components were actually not being used. Thus I delete them from code.

QA Steps:
- The PR only removes un-used code, a check can be done to ensure that gas tooltip on confirmation pages work as before.

![Screenshot 2023-04-19 at 2 18 32 PM](https://user-images.githubusercontent.com/2182307/233021630-2bf4bc3b-efe2-476d-8f3a-6172374f7460.png)
